### ISSUE TITLE
multi-scale input

### ISSUE THREAD

**Issue by ForawardStar** [pkuCactus/BDCN#39]
How to get the multi-scale input?

**Comment by DREAMXFAR** [comment]
@ForawardStar  you can refer to test_ms.py `test_img = Data(test_root, test_lst, mean_bgr=mean_bgr, scale=[0.5, 1, 1.5])`

**Comment by ForawardStar** [comment]
@DREAMXFAR  Thanks for your answer!! It solved my problem. I notice that BDCN are trained on augmentated BSDS500 and PASCAL VOC Context datasets, Could you please provide your training data to us for helping us to reproduce your result (ODS-F: 0.820). By the way, the paper says that the threshold used for loss computation is set as 0.3 for BSDS500,,but the 'yita' value in cfg.py is 0.5, is this a mistake？Looking forward to your reply.
